index.bs

```diff
@@ -10174,8 +10174,22 @@ registerProcessor('BitCrusher', class extends AudioWorkletProcessor {
 	process (inputs, outputs, parameters) {
 		let input = inputs[0];
 		let output = outputs[0];
+
```

Comment:
Nothing wrong with this code, but how it's done defeats the purpose of this feature completely. Not sure if this is "exemplary".

Reply:
I can certainly write a bunch of extra code to handle the 4 possible scenarios if you want.

Reply:
Using modulo operation can do the trick, I believe.

```js
...
let step = Math.pow(0.5, bitDepth[i % bitDepth.length]);
this._phase += frequencyReduction[i % frequencyReduction.length];
...
```